src/widgets/mentors/ui/mentors.test.tsx

```diff
@@ -24,7 +26,7 @@ describe('Mentors', () => {
     const button = screen.getByRole('link', { name: /Become a mentor/i });
 
     expect(button).toBeVisible();
-    expect(button).toHaveAttribute('href', 'https://app.rs.school/registry/mentor');
+    expect(button).toHaveAttribute('href', registerLink);
```

Comment:
I can't write comments for lines above - my suggestions:
- add test-case for checking the component is visible using data-testid
- combine test-cases on lines 14 & 19

Reply:
Done